app/views/tags/result_doc.scala.html

```diff
@@ -18,6 +18,8 @@
 
 @gndEntities = @{ 
 	Option(Seq(
+	    "label",
+	    "altLabel",
 	    "preferredName",
 	    "preferredNameForTheWork",
 	    "preferredNameForTheFamily",
```

Comment:
We still have a fallback to use API 1.x, will remove these with other code after the 2.0 launch.